back/app/services/projects_listed_scope_service.rb

```diff
@@ -0,0 +1,33 @@
+class ProjectsListedScopeService
+  def self.remove_unlisted_projects(projects_scope)
```

Comment:
At some point in a backend CDM it was agreed to always use instance methods for services and not class methods, even when no state is involved. But I don't think this choice really matters, it's mostly about consistency.